apiary.apib

```diff
@@ -6,6 +6,31 @@ OpenGrok RESTful API documentation. The following endpoints are accessible under
 
 Besides `/suggester` and `/search` endpoints, everything is accessible from `localhost` only.
 
+## Annotation [/annotation{?path}]
+
+### Get annotation for a file [GET]
+
++ Parameters
+  + path (string) - path of file, relative to source root
+
++ Response 200 (application/json)
+  + Body
+
+            [
+              {
+                "number": 1,
+                "revision": "e2736ff6",
+                "author": "Vladimir Kotal",
+                "desc": "changeset:&nbsp;e2736ff6&lt;br/&gt;summary:&nbsp;add&nbsp;microbadges&lt;br/&gt;user:&nbsp;Vladimir&nbsp;Kotal&nbsp;&#60;vlada@devnull.cz&#62;&lt;br/&gt;date:&nbsp;Fri&nbsp;Jun&nbsp;28&nbsp;21:33:41&nbsp;CEST&nbsp;2019"
+              },
```

Comment:
we also utilize the version number in all version of the file (like version 4 out of 22), can you include that as well?

Reply:
I don't understand. Could you rephrase or provide example ?

Reply:
<img width="90" alt="Snímek obrazovky 2020-02-27 v 17 19 50" src="https://user-images.githubusercontent.com/6997160/75463129-a4e1b800-5985-11ea-9d9d-676243827f84.png">

Reply:
Oh that. I never noticed it before :-)